diff --git a/commet/android/app/build.gradle b/commet/android/app/build.gradle
index 8897f07..bf609ea 100644
--- a/commet/android/app/build.gradle
+++ b/commet/android/app/build.gradle
@@ -23,7 +23,7 @@ if (flutterVersionName == null) {
 
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
-// apply plugin: 'com.google.gms.google-services'
+apply plugin: 'com.google.gms.google-services'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 def keystoreProperties = new Properties()
diff --git a/commet/android/build.gradle b/commet/android/build.gradle
index 7b03921..a059ce6 100644
--- a/commet/android/build.gradle
+++ b/commet/android/build.gradle
@@ -8,7 +8,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:7.2.0'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        // classpath 'com.google.gms:google-services:4.3.8'
+        classpath 'com.google.gms:google-services:4.3.8'
     }
 }
 
diff --git a/commet/lib/client/components/push_notification/firebase_push_notifier.dart b/commet/lib/client/components/push_notification/firebase_push_notifier.dart
index 67d69a7..beb0791 100644
--- a/commet/lib/client/components/push_notification/firebase_push_notifier.dart
+++ b/commet/lib/client/components/push_notification/firebase_push_notifier.dart
@@ -12,10 +12,10 @@ import 'package:commet/service/background_service.dart';
 import 'package:commet/service/background_service_notifications/background_service_task_notification.dart';
 
 // Manage these to enable / disable firebase
-// import 'package:firebase_core/firebase_core.dart';
-// import 'package:firebase_messaging/firebase_messaging.dart';
-dynamic Firebase;
-dynamic FirebaseMessaging;
+import 'package:firebase_core/firebase_core.dart';
+import 'package:firebase_messaging/firebase_messaging.dart';
+//dynamic Firebase;
+//dynamic FirebaseMessaging;
 // --------
 
 Future<void> onForegroundMessage(dynamic message) async {
diff --git a/commet/lib/config/build_config.dart b/commet/lib/config/build_config.dart
index 5134142..417a061 100644
--- a/commet/lib/config/build_config.dart
+++ b/commet/lib/config/build_config.dart
@@ -20,7 +20,7 @@ class BuildConfig {
       String.fromEnvironment('BUILD_DETAIL', defaultValue: "default");
 
   static const bool ENABLE_GOOGLE_SERVICES =
-      bool.fromEnvironment("ENABLE_GOOGLE_SERVICES", defaultValue: false);
+      bool.fromEnvironment("ENABLE_GOOGLE_SERVICES", defaultValue: true);
 
  static const String PUSH_GATEWAY =
      String.fromEnvironment('PUSH_GATEWAY', defaultValue: "unset");
diff --git a/commet/pubspec.yaml b/commet/pubspec.yaml
index 8163da0..e3bb1c5 100644
--- a/commet/pubspec.yaml
+++ b/commet/pubspec.yaml
@@ -88,8 +88,8 @@ dependencies:
   ## ---- Putting some extra lines in here to help git with the diff
   ## ---- Probably good to keep this stuff at the bottom of the list
   ## Begin Google Services Dependencies
-  #firebase_core: ^2.30.0
-  #firebase_messaging: ^14.7.10
+  firebase_core: ^2.30.0
+  firebase_messaging: ^14.7.10
   ## End Google Services Dependencies
   ## ----
   ## ----
